chapters/ch04/1.setting-up-the-retrotech-dataset.ipynb

```diff
@@ -12,15 +12,25 @@
    "cell_type": "code",
    "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<class 'engines.opensearch.OpenSearchEngine.OpenSearchEngine'>\n"
+     ]
+    }
+   ],
    "source": [
-    "import sys\n",
-    "sys.path.append('../..')\n",
-    "from aips import display_product_search, get_engine\n",
     "from pyspark.sql import SparkSession\n",
+    "\n",
+    "from aips import display_product_search, get_engine, set_engine\n",
     "from aips.spark.dataframe import from_csv\n",
+    "\n",
+    "set_engine(\"opensearch\")\n",
     "engine = get_engine()\n",
-    "spark = SparkSession.builder.appName(\"AIPS\").getOrCreate()"
+    "spark = SparkSession.builder.appName(\"AIPS\").getOrCreate()\n",
+    "print(type(engine))"
```

Comment:
Not sure if engine needs to be printed here.

Reply:
Removed